Hearthstone Deck Tracker/Hearthstone/Card.cs

```diff
@@ -115,7 +114,7 @@ public Card(HearthDb.Card dbCard)
 			Id = dbCard.Id;
 			Count = 1;
 			PlayerClass = HearthDbConverter.ConvertClass(dbCard.Class);
-			Rarity = HearthDbConverter.RariryConverter(dbCard.Rarity);
+             Rarity = dbCard.Rarity;

```

Comment:
I don't know what is wrong with the white space upon submitting a pull request. It looks fine in VS.